<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>displayScores</class>
 <widget class="QWidget" name="displayScores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="layout_results" stretch="1,1,1,4,1">
     <item>
      <widget class="QPushButton" name="button_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_legend" stretch="2,2,1">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Name">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Score">
         <property name="text">
          <string>Score</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Group">
         <property name="text">
          <string>Group</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="midLineWidth">
        <number>4</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_players" stretch="2,2,1">
       <item>
        <layout class="QVBoxLayout" name="layout_name"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layout_score"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layout_group"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_stats">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_mean">
         <property name="text">
          <string>Mean</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_stdDev">
         <property name="text">
          <string>StdDev</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
